controller/workitem.go

```diff
@@ -575,15 +608,17 @@ func ConvertWorkItem(request *http.Request, wi workitem.WorkItem, additional ...
 	workItemIncludeComments(request, &wi, op)
 	workItemIncludeChildren(request, &wi, op)
 	for _, add := range additional {
-		add(request, &wi, op)
+		if err := add(request, &wi, op); err != nil {
+			return nil, errs.Wrapf(err, "failed to run additional conversion function")
```

Comment:
There is format string & corresponding value(s) here. So, please use `errs.Wrap` instead.

Reply:
Done in 2a0afcc and e06f74d